docs/sql-reference/sql-statements/data-manipulation/STREAM LOAD.md

```diff
@@ -18,13 +18,13 @@ curl --location-trusted -u <username>:<password> -XPUT <url>
 
 This topic uses curl as an example to describe how to load data by using Stream Load. In addition to curl, you can also use other HTTP-compatible tools or languages to perform Stream Load. Load-related parameters are included in HTTP request header fields. When you input these parameters, take note of the following points:
 
-- You can use HTTP chunked transfer encoding. If you do not choose chunked encoding, you must input a `Content-Length` header field to indicate the length of content to be transferred, thereby ensuring data integrity.
+- You can use HTTP chunked transfer encoding, as demonstrated in this topic. If you do not choose chunked encoding, you must input a `Content-Length` header field to indicate the length of content to be transferred, thereby ensuring data integrity.
 
   > **NOTE**
   >
   > If you use curl to perform Stream Load, StarRocks automatically adds a `Content-Length` header field and you do not need manually input it.
 
-- We recommend that you add an `Expect` header field and specify its value as `100-continue`, as in `"Expect:100-continue"`. This helps prevent unnecessary data transfers and reduce resource overheads in case your job request is denied.
+- You must add an `Expect` header field and specify its value as `100-continue`, as in `"Expect:100-continue"`. This helps prevent unnecessary data transfers and reduce resource overheads in case your job request is denied.
```

Comment:
You must add an `Expect` header field in the HTTP request？？？

Reply:
No need I think. There's enough context to show where this, and the preceding content-length header field, should be added.